### ISSUE TITLE
Can i user IXHttpClient to upload file to server?

### ISSUE THREAD

**Issue by kkseven** [machinezone/IXWebSocket#480]
Hi, I'm new in IXWebSocket.

Our project needs to use websocket and http client. IXWebSocket works fine, in addition, we need to download some files from server 
using IXHttpClient. I can download files from server by setting args->onChunkCallback, and then it will be callback while receiving file datas.

But I could't find any methods or settings to download a file.

Did I missing something?

**Comment by bsergean** [comment]
Yes you can, usually you need to set the POST or PUT HTTP method to upload data.
